docs/F-Refinements/pointers-arrays-and-structs.md

```diff
@@ -389,5 +398,7 @@ Grade 3 : 7
 
 The array is stored in the ***stack*** segment of RAM **alongside the local variables**.
 
-> **!!! PORTABILITY ALERT !!!**<br/>
+:::important
+**!!! PORTABILITY ALERT !!!**<br/>
```

Comment:
Is it supposed to be in the next line?
And is all subheadings (**) are to be replaced with ":::"?
If so, it looks good!

Reply:
Hey Irene

> Is it supposed to be in the next line? And is all subheadings (**) are to be replaced with ":::"? If so, it looks good!

I am actually replacing the blockquotes with admonitions so ">" becomes ":::". The admonitions though only have certain keywords like important, caution, note etc. In this case, I thought important was pretty good while retaining the reason which is portability. Thanks for reviewing :)